modules/seqexec/web/client/src/test/scala/seqexec/web/client/model/ModelSpec.scala

```diff
@@ -38,6 +39,11 @@ final class ModelSpec extends CatsSuite with ArbitrariesWebClient {
 
   // lenses
   checkAll("Lens[SequenceTab, Option[Int]]", LensTests(SequenceTab.stepConfigL))
-  checkAll("Lens[SequenceTab, RefTo[Option[SequenceView]]]", LensTests(SequenceTab.currentSequenceL))
-  checkAll("Optional[SequenceTab, SequenceView]", OptionalTests(SequenceTab.completedSequenceO))
+
+  // Property generation errors
+  // checkAll("SeqexecTab.previewTab", PrismTests(SeqexecTab.previewTab))
+  // checkAll("SeqexecTab.instrumentTab", PrismTests(SeqexecTab.instrumentTab))
+  // checkAll("SeqexecTab.sequenceTab", PrismTests(SeqexecTab.sequenceTab))
+  // checkAll("SequencesOn.focusSequence", OptionalTests(SequencesOnDisplay.focusSequence))
+  // checkAll("SequencesOnDisplay.previewTab", TraversalTests(SequencesOnDisplay.previewTab))
```

Comment:
Why are these tests included, but commented?

Reply:
because they fail due to a `scalacheck` issue which i hope we can fix at some moment in time